about.html

```diff
@@ -0,0 +1,170 @@
+        </div>
+      </section>
+      <!-- CC logo inform and -->
+      <section id="logo-contest">
+        <div class="container d-flex flex-column align-items-center">
+          <h2 class="my-4">Indie Gaming Community 2022 Logo</h2>
+          <p class="text-center w-md-75">
+            The logo of Indie Gaming Community 2022 was decided through the logo
+            competition from 8. June to 7. July. After online public voting and
+            Selection Committee's judging process, the logo from Naresh Agrawal
+            won the prize. After consultation with Mr. Agrawal we optimized the
+            prize-winning-logo for this year's summit as following:
+          </p>
+          <div>
+            <img
+              class="text-center"
+              src="images/CCGSlogo-long.png"
+              alt="Indie Gaming Community 2022 Winning Logo"
+            />
+          </div>
+        </div>
+      </section>
+      <!-- Past Summits -->
+      <section id="past-summits" class="pb-4">
+        <div
+          class="container d-flex border-top pt-3 flex-column align-items-center"
+        >
+          <h2>See the past Indie Gaming Community 2022 gatherings</h2>
+          <p class="py-3">
+            Take a look at the last two Indie Gaming Community 2022 gatherings
+            which took place in Buenos Aires and in Warsaw.
+          </p>
+          <div
+            class="flex-row d-flex gap-2 flex-md-row flex-column justify-content-center"
+          >
+            <div class="about-card-wrapper">
+              <h3 class="fw-bold text-light">2013</h3>
+              <p class="text-light">
+                IG Community 2013 gathering in Buenos Aires
+              </p>
+            </div>
+            <div class="about-card-wrapper">
+              <h3 class="fw-bold text-light">2011</h3>
+              <p class="text-light">IG Community 2011 gathering in Warsaw</p>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!-- Partners Section -->
+      <section id="partners" class="bg-dark">
+        <div class="container">
+          <h2 class="fw-bold text-center text-secondary py-3">Partners</h2>
+          <ul
+            class="list-unstyled pb-3 text-secondary d-flex flex-wrap justify-content-center align-items-center gap-4"
+          >
+            <li class="fw-bold">Mozilla</li>
+            <li>Google</li>
+            <li>NAVER</li>
+            <li>Daumkakao</li>
+            <li><img src="/images/airbnb-brands.svg" alt="" /> airbnb</li>
+          </ul>
+        </div>
+      </section>
+      <!-- About Footer Section -->
+      <footer
+        class="d-flex align-items-center justify-content-center container"
+      >
+        <img
+          src="images/CreativeCommons_logo_trademark.svg"
+          alt="CC trademark logo"
+        />
+        <p class="ps-3 mb-0">
+          2022 Creative Commons <br />
+          Some Rights Reserved.
+        </p>
+      </footer>
```

Comment:
- [ ] Kindly make the image smaller so you can give the text more space, to make your design much better.

![image](https://user-images.githubusercontent.com/35535130/181618407-b1be29ff-2761-423e-90e2-1f954c474d55.png)